tests/test_pyright.py

```diff
@@ -20,9 +21,11 @@ class PyrightDiagnostic:
 
 
 @pytest.mark.skipif(not _found_pyright, reason="Requires pyright.")
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="Requires Python 3.7+")
```

Comment:
I had to add this because subprocess uses 3.7 and this tests breaks if there's a global pyright.